mappings/net/minecraft/client/settings/GameSettings.mapping

```diff
@@ -13,9 +13,14 @@ CLASS none/bdt net/minecraft/client/settings/GameSettings
 		FIELD K SHOW_SUBTITLES Lnone/bdt$a;
 		FIELD L REALMS_NOTIFICATIONS Lnone/bdt$a;
 		FIELD M AUTO_JUMP Lnone/bdt$a;
+		FIELD N isSlider Z
+		FIELD O isBoolean Z
```

Comment:
I was considering `toggle` but it didn't feel quite right. I guess I ended up prefixing these because what they indicate is somewhat confusing if they are just `slider` and `switch`. Being able to doc this would be nice :)